### PR TITLE
DBZ-3413 Handle double single quotes in LogMinerDmlParser

### PR DESCRIPTION
**Pull request by infinitesnowboy**
https://issues.redhat.com/browse/DBZ-3413

Double single quotes are treated as 2 quotes instead of 1 escaped quote in the WHERE clause parsing function. Small change to fix this and added a unit test.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Thanks, @infinitesnowboy!

@Naros, could you review this one?

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java [hunk 1]
```diff
@@ -336,4 +336,38 @@ public void shouldParsingRedoSqlWithParenthesisInFunctionArgumentStrings() throw
         assertThat(entry.getOldValues().get(1).getColumnData()).isNull();
         assertThat(entry.getNewValues()).isEmpty();
     }
+
+    @Test
+    @FixFor("DBZ-3413")
+    public void testParsingDoubleSingleQuoteInWhereClause() throws Exception {
```
Comment (Naros) [inline debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java:342]: Could we either add tests or adjust this specific test to verify this double single quote behavior in both an `INSERT` statement's column value as well as in an `UPDATE` statement's set-clause for completeness?

**Review by Naros — CHANGES_REQUESTED** [review]

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java (hunk 1 above)
Comment (infinitesnowboy) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java:342]: Sure, I added those test cases into this test.

**Comment by gunnarmorling** [comment]
Thx for the update, @infinitesnowboy! @Naros, is this good to go with this change?

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java (hunk 1 above)
Comment (Naros) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerDmlParserTest.java:342]: Thanks!

**Review by Naros — APPROVED** [review]

**Comment by Naros** [comment]
Merged, thanks @infinitesnowboy!

**Pull request merged**